pkg/utils/validation/admissionplugins/admissionplugins.go

```diff
@@ -147,8 +158,11 @@ func ValidateAdmissionPlugins(admissionPlugins []core.AdmissionPlugin, version s
 		} else if !supported {
 			allErrs = append(allErrs, field.Forbidden(idxPath.Child("name"), fmt.Sprintf("admission plugin %q is not supported in Kubernetes version %s", plugin.Name, version)))
 		} else {
+			if admissionPluginHasConfig(plugin) && !admissionPluginAllowsConfiguration(plugin) {
+				allErrs = append(allErrs, field.Forbidden(idxPath.Child("config"), fmt.Sprintf("admission plugin %q does not allow configuration", plugin.Name)))
+			}
 			if admissionPluginsVersionRanges[plugin.Name].Forbidden {
-				allErrs = append(allErrs, field.Forbidden(idxPath.Child("name"), fmt.Sprintf("forbidden admission plugin was specified - do not use plugins from the following list: %+v", getAllForbiddenPlugins())))
+				allErrs = append(allErrs, field.Forbidden(idxPath.Child("name"), fmt.Sprintf("forbidden admission plugin was specified - do not use plugins from the following list for Kubernetes version %s: %+v", version, getAllForbiddenPlugins(version))))
```

Comment:
Maybe only get the list once in case multiple disallowed plugins are configured?